src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(./fonts/Avenir.ttc);
+}
+
+body {
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+}
+
+.search-bar__wrapper {
+  position: relative;
+  margin-top: 20px;
+}
+
+.search-bar__form {
+  position: relative;
+  display: flex;
+  text-align: center;
+}
+
+.search-bar__label {
+  position: absolute;
+}
+
+.search-bar__label.big {
+  top: 25px;
+  left: 26px;
+  height: 19px;
+  width: 19px;
+}
+
+.search-bar__label.small {
+  top: 12px;
+  left: 14px;
+  height: 11px;
+  width: 11px;
+}
+
+.search-bar__img {
+  height: 100%;
+  width: 100%;
+}
+
+.search-bar__input {
+  color: #3d4e61;
+  width: 100%;
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  transition: box-shadow 0.3s, background 0.8s;
+}
+
+.search-bar__input.big {
+  font-size: 1rem;
+  height: 70px;
+  padding-left: 60px;
+  padding-right: 20px;
+}
+
+.search-bar__input.small {
+  font-size: 0.9rem;
+  height: 42px;
+  padding-left: 34px;
+  padding-right: 15px;
+}
+
+.search-bar__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-bar__input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-family: Avenir, sans-serif;
```

Comment:
You already have this for main state
```suggestion
```

Reply:
Done. I remove all repeated 'font-style' Left only for Input element.